test/cppsuite/tests/poc.cxx

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+extern "C" {
+    #include "wiredtiger.h"
+}
+
+int main(int argc, char *argv[]) {
+    WT_CONNECTION *conn;
+    int ret = 0;
+    // Setup basic test directory.
+    const std::string default_dir = "WT_TEST";
+
+    // Csuite tests utilise a test_util.h command to make their directory, currently that doesn't
```

Comment:
Is the intention here to use the _C++ style_ comments. I would suggest we continue using the coding conventions we have for the rest of the wired tiger code.

Reply:
Oh duh, yeah I'll fix that up.